tests/unit/data_access/test_entity_queries.py

```diff
@@ -1,9 +1,9 @@
+from datetime import datetime
 import pytest
```

Comment:
FYI I'd say this probably qualifies as integration testing now because you aren't mocking the inner workings of the function, so could move these tests to the integration folder